chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -68,6 +76,16 @@ public int put(BlockSpec spec) {
     return id;
   }
 
+  public void updateProperties(String name, Consumer<Block> properties) {
+    //TODO definitely needs some optimizing
+    materialProperties.put(name, properties);
+    blockMap.forEach((spec, id) -> {
+      Block block = spec.toBlock();
+      applyMaterial(block);
+      palette.set(id, block);
+    });
```

Comment:
Hmmm, this doesn't seem very efficient but i don't think there is a way to get a BlockSpec from the name (to get the id via the blockMap), let's see what leMaik thinks

Reply:
It's only done on scene load and it's ~800 blocks, shouldn't be too significant.